src/NHSD.BuyingCatalogue.Ordering.Services/NHSD.BuyingCatalogue.Ordering.Services.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>D:\Project\GPITFutures\API\13236_ORDAPI\BuyingCatalogueOrdering\NHSD.BuyingCatalogue.Ordering.Services\NHSD.BuyingCatalogue.Ordering.Services.xml</DocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\NHSD.BuyingCatalogue.Ordering.Contracts\NHSD.BuyingCatalogue.Ordering.Contracts.csproj" />
+    <ProjectReference Include="..\NHSD.BuyingCatalogue.Ordering.Domain\NHSD.BuyingCatalogue.Ordering.Domain.csproj" />
+    <ProjectReference Include="..\NHSD.BuyingCatalogue.Ordering.Persistence\NHSD.BuyingCatalogue.Ordering.Persistence.csproj" />
+  </ItemGroup>
+
+</Project>
```

Comment:
We treat warning as errors in release builds so can you enable that for the release build? Also make sure the release build also has the element below.

```xml
<GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Reply:
Also please make sure to install the latest pre-release version of StyleCop.

```powershell
Install-Package StyleCop.Analyzers -Version 1.2.0-beta.333
```

Reply:
Updated Warning for release builds and added `StyleCop.Analyzers` package